user/v1/events/added.avsc

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "record",
+  "name":"UserAdded",
+  "namespace":"io.renku.events.v1",
+  "doc":"Event raised when a new user is added",
+  "fields":[
+    {
+      "name":"id",
+      "type":"string",
+    },
+    {
+      "name": "first_name",
```

Comment:
Could we maybe keep the camelCase as in the `ProjectCreated`, please?

Reply:
right. getting carried away with python :laughing: